cdap-ui/app/cdap/components/FastAction/SetPreferenceAction/SetPreferenceModal.js

```diff
@@ -403,20 +402,20 @@ export default class SetPreferenceModal extends Component {
                       className="btn btn-primary float-xs-left"
                       disabled="disabled"
                     >
-                      <span className="fa fa-spinner fa-spin"></span>
+                      <span className="fa fa-spinner fa-spin" />
```

Comment:
It's not HTML, but JSX, so you can use self-closing tags if there are no children:

https://facebook.github.io/react/docs/jsx-in-depth.html